pmacApp/Db/pmacController.template

```diff
@@ -229,13 +229,70 @@ record(mbbi, "$(P):PMACTYPE") {
   field(INP, "@asyn($(PORT),0)PMAC_VIM_cid")
 }
 
+# pmac number of CPU cores
+record(ai, "$(P):CPUNUMCORES") {
+  field(DESC, "Number of CPU cores")
+  field(DTYP, "asynInt32")
+  field(SCAN, "I/O Intr")
+  field(INP, "@asyn($(PORT),0)PMAC_C_CPU_NUM_CORES")
+  field(HIGH, "4")
```

Comment:
Do we want severity on this record?  Wouldn't it always be in an alarm state if the alarm is based on number of CPU cores?

Reply:
The alarms were implemented to detect at the EPICS level if reading the number of cores was not possible, or if the number of cores exceeded the currently supported limit of 4 cores. However, upon further reflection, it seems more appropriate to allow the driver to handle this entirely.